packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -154,10 +279,11 @@ class MethodChannel {
   @optionalTypeArgs
   Future<T?> _invokeMethod<T>(String method, { required bool missingOk, dynamic arguments }) async {
     assert(method != null);
-    final ByteData? result = await binaryMessenger.send(
-      name,
-      codec.encodeMethodCall(MethodCall(method, arguments)),
-    );
+    final ByteData input = codec.encodeMethodCall(MethodCall(method, arguments));
+    final ByteData? result =
+      !kReleaseMode && debugProfilePlatformChannels ?
+        await (binaryMessenger as _ProfiledBinaryMessenger).sendWithPostfix(name, '#$method', input) :
```

Comment:
Yea I chose `#` as a delimiter since `.` and `/` are typically used in channel names and I wanted a clear distinction between a channel name and a method.